src/consts/urls.ts

```diff
@@ -109,3 +109,11 @@ export const getTunnelingDomainUrl = (): string => {
 export const removeAppStorageDataForAccountUrl = (appId: AppId, targetAccountId: AccountId): string => {
   return `${BASE_APPS_URL}/${appId}/accounts/${targetAccountId}`;
 };
+
+export const importAppManifestUrl = (): string => {
+  return `${BASE_APPS_URL}/manifest`;
+};
+
+export const exportAppManifestUrl = (appId: AppId): string => {
+  return `${BASE_APPS_URL}/${appId}/manifest?zipBase64=true`;
```

Comment:
why do you need the `zipBase64=true`?

Reply:
The default is getting the zip in the response as buffer, which is harder to "catch" it programatically.
Here we getting it as base64 string and we can easily dump it to a file